worker/codalabworker/local_run/local_run_manager.py

```diff
@@ -360,3 +360,26 @@ def memory_bytes(self):
         except ValueError:
             # Fallback to sysctl when os.sysconf('SC_PHYS_PAGES') fails on OS X
             return int(check_output(['sysctl', '-n', 'hw.memsize']).strip())
+
+    @property
+    def free_disk_bytes(self):
+        """
+        Available disk space by bytes of this RunManager.
+        """
+        try:
+            p = Popen(["df", self._work_dir], stdout=PIPE)
+            output, error = p.communicate()
+            # Return -1 when there is an error.
+            if error:
+                logger.info(error.strip())
+                return -1
```

Comment:
Hm, returning `-1` doesn't seem great because the semantics of an integer return value become more complex.  Maybe `None` is a bit better?

Reply:
I also updated value of `nullable` in column definition and alembic revision file to be `True`.